houston/ardu/sandbox.py

```diff
@@ -105,6 +105,9 @@ def start(self,
         the vehicle running inside the simulation. Blocks until SITL is
         launched and a connection is established.
         """
+        # TODO make it optional to compile with coverage
```

Comment:
Suggestion: Create an issue for this `TODO` and link it to the comment. For example:
```
# TODO make it optional to compile with coverage (#XXX)
```